core/iwasm/interpreter/wasm_runtime.c

```diff
@@ -2560,20 +2560,94 @@ wasm_interp_dump_call_stack(struct WASMExecEnv *exec_env)
             }
         }
 
+        frame.func_name_wp = func_name;
+
+        bh_vector_append(module_inst->frames, &frame);
```

Comment:
Should check return value, if false, destroy module_inst->frames, and set it to NULL?

Reply:
Done